JSDemos/Demos/TagBox/Overview/description.md

```diff
@@ -1 +1,36 @@
-The TagBox component is&nbsp;an&nbsp;editor that allows an&nbsp;end user to&nbsp;select multiple items from a&nbsp;drop-down list.
\ No newline at end of file
+The [TagBox](/Documentation/ApiReference/UI_Components/dxTagBox/) allows users to select multiple items from a drop-down list. This demo illustrates the following TagBox properties:
+
+- [items](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/items/)  
+  An array of items displayed by the TagBox.
+
+- [searchEnabled](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#searchEnabled)  
+  Specifies whether the search is enabled
+
+- [showSelectionControls](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#showSelectionControls)  
+  Defines whether selection controls are displayed
+
+- [applyValueMode](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#applyValueMode)  
+  Specifies whether selected values are applied instantly or when a user clicks the **OK** button.
+
+- [hideSelectedItems](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#hideSelectedItems)  
+  Specifies whether items are removed from the drop-down list upon selection.
+
+- [multiline](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#multiline)  
+  Displays selected items on single or multiple lines.
+
+- [acceptCustomValue](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#acceptCustomValue)  
+  Specifies whether users can enter custom values. Requires the [onCustomItemCreating](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#onCustomItemCreating) implementation.
+
+- [placeholder](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#placeholder)  
+  Customizes the TagBox placeholder.
+
+- [value](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#value)  
+  Specifies items the TagBox displays.
+
+- [disabled](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#disabled)  
+  Defines whether the component resopnds to user interaction.
+
+- [dataSource](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#dataSource)  
+  Binds the TagBox to data. Unlike the **items** property, **dataSource** accepts the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object that allows users to sort, filter, group, and shape data.
+- [itemTemplate](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#itemTemplate)  
+  Allows you to customize the text field and drop-down list items.
```

Comment:
The [fieldTemplate](https://js.devexpress.com/Documentation/21_1/ApiReference/UI_Components/dxTagBox/Configuration/#fieldTemplate) property allows users to customize the text field.

Reply:
itemTemplate is used in this demo https://js.devexpress.com/Demos/WidgetsGallery/Demo/TagBox/Overview/jQuery/Light/

Reply:
Right you are. But does it customize the text field?